README.md

```diff
@@ -0,0 +1,44 @@
+# InterviewBot
+
+
+
+
+InterviewBot - is a Python program for interviewing candidates. It allows you to create questions, record candidates' answers and save interview results.
+
+## Installation
+
+First clone this repository to your computer:
+
+```
+git clone https://github.com/AntonAks/InterviewBot.git
+```
+
+Starts docker-compose:
+```
+docker-compose up -d
+```
+
+
+Go to the project directory:
+```
+cd InterviewBot
+```
+
+Activate your Poetry virtual environment:
+```
+poetry shell
+```
+
+Install dependencies:
+```
+poetry install fastapi, uvicorn, asyncpg,
+```
+
+## Using
+Run the main program file:
+```
+python main.py
+```
```

Comment:
1. Do we need this (22-41 rows) if we run the application using `docker-compose` commands?
2. Add please how to run tests.